latest/index.bs

```diff
@@ -18,12 +17,12 @@ Editor: Sébastien Besson, Open Microscopy Environment (OME) https://www.openmic
 Abstract: This document contains next-generation file format (NGFF)
 Abstract: specifications for storing bioimaging data in the cloud.
 Abstract: All specifications are submitted to the https://image.sc community for review.
-Status Text: The current released version of this specification is
-Status Text: <a href="../0.1/index.html">0.1</a>. Migration scripts
```

Comment:
This text should stay as is but point to the `0.3` that you will create when `latest/index.bs` is done.